packages/ag-charts-community/src/scene/shape/text.ts

```diff
@@ -200,8 +202,11 @@ export class Text extends Shape {
         maxWidth: number,
         maxHeight: number,
         textProps: TextSizeProperties,
-        wrapping: TextWrap
+        wrapping: TextWrap,
+        // This API isn't finished - for now this function will return an empty string if overflow is `never` and the text overflows
+        overflow: TextOverflow = 'ellipsis'
```

Comment:
Nit: Can we remove the need for this comment - either by adding the required behaviour, or narrowing the API to explicitly exclude the unsupported cases?